pootle/apps/pootle_project/models.py

```diff
@@ -224,8 +228,11 @@ def accessible_by_user(cls, user):
 
         :param user: The ``User`` instance to get accessible projects for.
         """
-        username = 'nobody' if user.is_anonymous() else user.username
-        key = iri_to_uri('projects:accessible:%s' % username)
+        if user.is_superuser:
+            key = iri_to_uri('projects:all')
+        else:
+            username = 'nobody' if user.is_anonymous() else user.username
```

Comment:
Not critical now but I realize `user.username` can directly be used since #4509, so no need to check for anonymous.